pypeit/core/flux_calib.py

```diff
@@ -139,6 +133,10 @@ def find_standard_file(ra, dec, toler=20.*units.arcmin, check=False):
                 std_dict['wave'] = std_spec['col1'] * units.AA
                 std_dict['flux'] = std_spec['col2']*1e-16/PYPEIT_FLUX_SCALE * \
                                    units.erg / units.s / units.cm ** 2 / units.AA
+                # At this low resolution, best to throw out entries affected by A and B-band absorption
```

Comment:
I'm not following why you are masking the standard here, whereas we have a routine that basically masks Balmer lines, etc.

Reply:
The default masking was working poorly for these ~50A binned ESO standards.